atd-vze/src/views/Crashes/Recommendations.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import { useQuery, useMutation } from "@apollo/react-hooks";
+import { Card, CardHeader, CardBody, CardFooter, Table, Button } from "reactstrap";
+import { recommendationsDataMap } from "./recommendationsDataMap";
+import { GET_RECOMMENDATIONS } from "../../queries/recommendations";
+import { useAuth0, isReadOnly } from "../../auth/authContext";
+
+// declare fatality review board recommendations component
+const Recommendations = ({ crashId }) => {
+
+  // fetch data from database using graphQL query
+  const { loading, error, data, refetch } = useQuery(GET_RECOMMENDATIONS, {
+    variables: { crashId },
+  });
+
+  if (loading) return "Loading...";
+  if (error) return `Error! ${error.message}`;
+
+  // if crash has data in recommendations table, display it
+  const displayData = (field) => {
+    if (data.recommendations[0]) {
```

Comment:
since you access it a lot, i suggest you assign `data.recommendations[0]` to a variable (as you do with `fieldConfig`)

Reply:
...and if there's a chance that `data` will be `undefined`, or that it will not have any recommendations, use nullish chaining

```
const rec = data?.recommendations?.[0]
```

Reply:
good call, definitely will assign it to a variable and use nullish chaining